### PR TITLE
Nuke Variant, AltAllele

### PR DESCRIPTION
**Pull request by cseed**
Builds on: https://github.com/hail-is/hail/pull/3268

Well, moved the to the tests section.  The test section is becoming a real cesspool.

### CODE REVIEW

**Comment by jbloom22** [comment]
please rebase now that #3268 is in

**Comment by cseed** [comment]
Rebased.

File: src/main/scala/is/hail/methods/SplitMulti.scala [hunk 1]
```diff
@@ -87,23 +87,23 @@ class SplitMultiPartitionContextIR(
...
+  def constructSplitRow(splitVariants: Iterator[(Locus, IndexedSeq[String], Int)], rv: RegionValue, wasSplit: Boolean): Iterator[RegionValue] = {
     if (!globalsCopied)
       copyGlobals()
     splitRegion.clear(globalsEnd)
     rvb.start(matrixType.rvRowType)
     rvb.addRegionValue(matrixType.rvRowType, rv)
     val oldRow = rvb.end()
     val oldEnd = splitRegion.size
-    splitVariants.map { case (sjv, aIndex) =>
+    splitVariants.map { case (svjLocus, svjAlleles, aIndex) =>
```
Comment (jbloom22) [inline src/main/scala/is/hail/methods/SplitMulti.scala]: `sv` stands for split variant, right? I don't fine `j` helpful for readability here or in 221 below. And on line 130 you use `newLocus` and `newAlleles` for the same concept. (not in scope, but I'd also change `splitrv` to `splitRV`)

File: src/main/scala/is/hail/variant/AltAllele.scala [hunk 2]
```diff
@@ -1,10 +1,6 @@
 package is.hail.variant
```
Comment (jbloom22) [inline src/main/scala/is/hail/variant/AltAllele.scala]: As with `VariantMethods`, this file should now be called `AltAlleleMethods`

File: src/test/scala/is/hail/io/ExportVCFSuite.scala [hunk 3]
```diff
@@ -40,13 +40,13 @@ class ExportVCFSuite extends SparkSuite {
 
     val vdsNew = hc.importVCF(outFile, nPartitions = Some(10))
 
-    implicit val variantOrd = vdsNew.referenceGenome.variantOrdering
+    implicit val locusAlllesOrdering = vdsNew.rowKeyStruct.ordering.toOrdering
```
Comment (jbloom22) [inline src/test/scala/is/hail/io/ExportVCFSuite.scala]: locusAll[e]lesOrdering

File: src/main/scala/is/hail/methods/SplitMulti.scala [hunk 4]
```diff
@@ -224,10 +218,10 @@ abstract class SplitMultiPartitionContext(
     }
 
     if (sortAlleles)
-      splitVariants = splitVariants.sortBy { case (svj, i) => svj }(variantOrdering)
+      splitVariants = splitVariants.sortBy { case (svjLocus, svjAlleles, i) => Annotation(svjLocus, svjAlleles) }(locusAllelesOrdering.toOrdering)
```
Comment (jbloom22) [inline src/main/scala/is/hail/methods/SplitMulti.scala]: Also change this to newLocus, newAlleles

**Pull request merged**